### ISSUE TITLE
Vertex Texture Fetch not implemented on OpenGL Platform

### ISSUE THREAD

**Issue by QuQuasar** [MonoGame/MonoGame#6379]
Vertex Texture Fetch is supported on the DirectX platform, and in XNA 4.0. It is not supported on the OpenGL Platform, meaning features that require Vertex Texture Fetch to be implemented cannot be ported to the Mac and Linux platforms. 

Originally raised here: http://community.monogame.net/t/vertex-texture-fetch-in-an-opengl-project/10481/4

**To reproduce:**

Attempt to use the tex2d() or tex2dlod() function to retrieve texture data in any vertex shader on the OpenGL platform.

#### What version of MonoGame does the bug occur on:
- MonoGame 3.6

#### What operating system are you using:
- Windows

#### What MonoGame platform are you using:

- DesktopGL

**Comment by QuQuasar** [comment]
Noting that this issue is still present in Monogame 3.7.1.

**Comment by Jjagg** [comment]
Yes, when it does get implemented you can expect to see a discussion here.

**Comment by willmotil** [comment]
Wanted to add a usage case for this.

This would be a big speed gain for particle instancing as you could offload a lot of processing from the cpu to the gpu were it could speed things up considerably.

**Comment by willmotil** [comment]
#7352  claims to have fixed this in max's pull request for geometry shader support.


https://github.com/MonoGame/MonoGame/issues/4567

**Comment by orosbogdan** [comment]
Any updates on this ?
